hydra-node/test/Hydra/Model.hs

```diff
@@ -555,7 +555,12 @@ performCommit parties party paymentUTxO = do
               , let vk = getVerificationKey sk
               , let txOut = TxOut (mkVkAddress testNetworkId vk) val TxOutDatumNone ReferenceScriptNone
               ]
-      party `sendsInput` Input.Commit{Input.utxo = realUTxO}
+      forM_ nodes $ \node ->
+        lift . injectChainEvent node $
+          Observation
+            { observedTx = OnCommitTx{party, committed = realUTxO}
+            , newChainState = initialChainState
+            }
```

Comment:
Should: use `simulateCommit` instead

This is requiring maybe some refactoring, but the Model spec code should follow the Behavior spec. Also, it would be an ill-formed chain event with the new chain state being not consistent (`initialChainState` creates an `Idle` chain state)